03-loops-R.md

```diff
@@ -498,12 +498,53 @@ Sure enough, the maxima of these data sets show exactly the same ramp as the fir
 > [lesson](03-supp-loops-in-depth.html) that demonstrates how to properly repeat
 > operations in R.
 
+
+> ## Challenge - Review loops element {.challenge}
+>
+> Consider a vector with different words:
+>     ~~~
+>     words <- c('Just', 'an', 'example','for', 'class') 
+>     ~~~
+>
+> Which construction is a correct formulation of a loop to print TRUE if the 
+> number of letters in a word correspond to the position of the word in the 
+> vector?  e.g. words[1] should have length 1.
```

Comment:
What's a real-life situation where you'd want to do something like this? I.e., what's a motivating analogy?